app/views/layouts/_header.html.erb

```diff
@@ -1,11 +1,16 @@
-<header class="navbar navbar-fixed-top navbar-inverse">
+<header class="navbar navbar-fixed-top">
   <div class="container">
-    <%= link_to "Inicio", '#', id: "logo" %>
-    <nav>
+    <%= link_to "Inicio", root_path, id: "logo" %>
+    <nav class="barra">
+      <ul class="nav navbar-nav">
+        <li class="link"><%= link_to "Publicaciones", root_path,id:"link" %></li>
```

Comment:
If you dont have the link. Use # instead

Reply:
x2